qulice-checkstyle/src/test/resources/com/qulice/checkstyle/InvalidDiamondsUsage.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Hello.
+ */
+package foo.bar;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Better to use diamond operator where possible.
+ *
+ * @author John Smith (john@example.com)
+ * @version $Id$
+ * @since 1.0
+ */
+public final class InvalidDiamondsUsage {
```

Comment:
@vkuchyn It might sound contradictory to one of my comments from previous review, but this test should be split by two 
- one to check that error appears when we expect it (no true negative)
- another to check that no errors when we don't expect them (no false positive)